cljfmt/src/cljfmt/main.clj

```diff
@@ -156,7 +156,9 @@
       (assoc :indents (merge (:indents default-options)
                              (:indents options {})))))
 
-(def default-paths ["src" "test" "project.clj"])
+(def default-paths
+  (let [exists? (fn [path] (.exists (io/as-file path)))]
+    (filter exists? ["src" "test" "project.clj" "deps.edn"])))
```

Comment:
This needs to be in`-main`, not part of the `default-paths` definition. You can't be sure that this def is evaluated at the same time as the `-main` function is executed.

Reply:
Oh, interesting. So if I grok that right, we could check existence at process startup, the world shifts out from under us, and by the time our check/fix gets invoked the existence pattern might have changed. Yep, will fix.

Reply:
No, worse than that. The filter is applied when the file is compiled. With AOT, compilation may take place at an entirely different time, and on an entirely different machine, than execution.

Reply:
Hadn't thought of that. Thanks.